spec/models/general_info_spec.rb

```diff
@@ -1,42 +1,40 @@
 require 'rails_helper'
 
-describe '[General Info]', type: :model do
+describe GeneralInfo, type: :model do
   describe '[Validation]' do
-    before :each do
-      @info = create :general_info
-    end
+    let(:gi) { build :general_info }
```

Comment:
It just returns the same reference to the same object, so I can't redefine `gi` but I can work with it the way I do.